lib/game.rb

```diff
@@ -0,0 +1,183 @@
+# no structure yet, just drafting some logic
+# will split up into dif methods in the future
```

Comment:
You mention making a selection in your initial comment when you open the PR.  I might consider keeping comments like these there.  That way, your collaborators can find then in a central location and they don't sneak into your code.